spec/services/request_create_service_spec.rb

```diff
@@ -16,15 +16,15 @@
     end
   end
 
-  describe '#create' do
+  context 'simulate human approvals' do
```

Comment:
To my understanding, this and next context are both auto approval test cases, one is with normally created workflow, the other uses seed-generated workflow. I feel `human approval` is bit of misleading here.